README.md

```diff
@@ -38,9 +38,7 @@ source venv/bin/activate
 
 Once the virtual environment is activated, install the required dependencies in the requirements.txt file:
 ```bash
-  pip install PyQt5
```

Comment:
Remove the 3 statments 
pip install PyQt5
  pip install sys
  pip install sqlite3